theforeman.org/scripts/debian/stage_packages.sh

```diff
@@ -7,19 +7,41 @@ set -xe
 # * the web node must have the freight class applied
 
 # Find the deps in the build-dir from the previous step
-DEB_PATH=./debian/${os}/build-${project}
+DEB_PATH=$(find ./ -name build-${project})
 
 # Upload all builds to stagingdeb for testing
-echo "scratch build: uploading to stagingdeb/${os}/${repoowner}-${version}"
+HOSTS="web01.osuosl.theforeman.org"
 export RSYNC_RSH="ssh -i /home/jenkins/workspace/staging_key/rsync_freightstage_key"
 USER=freightstage
-HOSTS="web01.osuosl"
-COMPONENT=${repoowner}-${version}
+
+if [[ "$DEB_PATH" == *"plugins"* ]];then
+  # Don't upload scratch builds to deb.tf.o
```

Comment:
I think this comment is (now) misleading, as this hunk actually *does* upload to deb.tfm.o.

Maybe something like this: Plugins aren't staged. Builds from the main (theforeman) repo are uploaded directly to deb.tfm.o. Only user (PR) builds are uploaded to stagingdeb.tfm.o.